packages/contents/src/emscripten.ts

```diff
@@ -57,22 +57,39 @@ export interface IEmscriptenStream {
   position: number;
 }
 
+export function instanceOfStream(
+  nodeOrStream: IEmscriptenFSNode | IEmscriptenStream,
+): nodeOrStream is IEmscriptenStream {
+  return 'node' in nodeOrStream;
+}
+
 export interface IEmscriptenNodeOps {
-  getattr(node: IEmscriptenFSNode): IStats;
-  setattr(node: IEmscriptenFSNode, attr: IStats): void;
-  lookup(parent: IEmscriptenFSNode, name: string): IEmscriptenFSNode;
+  getattr(node: IEmscriptenFSNode | IEmscriptenStream): IStats;
```

Comment:
Looks like this change of signature can be considered as a breaking change?

Although it's probably not directly used that much downstream, should we still document it in the migration guide? https://jupyterlite.readthedocs.io/en/latest/migration.html